src/util/XDRStream.h

```diff
@@ -118,10 +136,78 @@ class XDRInputFileStream
         {
             throw xdr::xdr_runtime_error("malformed XDR file");
         }
+
         xdr::xdr_get g(mBuf.data(), mBuf.data() + sz);
         xdr::xdr_argpack_archive(g, out);
         return true;
     }
+
+    template <typename T>
+    bool
+    readPage(T& out, LedgerKey const& key, size_t pageSize)
```

Comment:
A comment here would be good, explaining what `readPage` is doing and what its parameters mean.